backend/src/services/conflict_service.py

```
@@ -0,0 +1,700 @@
+
+    @staticmethod
+    def _score_performer_lineup(event: Event, performer_ceiling: int = 5, **kwargs) -> float:
+        """Confirmed performers / ceiling * 100."""
+        confirmed = sum(
+            1 for ep in event.event_performers
+            if ep.status == "confirmed"
+        )
+        return min(confirmed / max(performer_ceiling, 1), 1.0) * 100
+
+    @staticmethod
+    def _score_logistics_ease(event: Event, **kwargs) -> float:
```

Ruff (0.15.0)

[warning] Unused static method argument: `kwargs`

(ARG004)